examples/deferred_processing/automated_deferred_processing/Makefile

```diff
@@ -0,0 +1,83 @@
+
+# The executable paths below are set to generic values.
+# Modify them for your system by setting environment variables, using a command like this
+# to both fetch the version of pachyderm you want to use and execute it.
+# "../etc/fetch_release_pachctl.py 1.10.0 ; env PACHCTL=${GOPATH}/bin/pachctl make -e opencv"
+
+SHELL := /bin/bash
+PACHCTL := pachctl
+KUBECTL := kubectl
+DOCKER := docker
+
+DEFERRED_PROCESSING_PIPELINES = ../deferred_processing_plus_transactions/edges_dp.json \
+				../deferred_processing_plus_transactions/montage_dp.json 
+
+DEFERRED_PROCESSING_1_FILES = ../deferred_processing_plus_transactions/images.txt \
+				../deferred_processing_plus_transactions/images2.txt
+
+DEFERRED_PROCESSING_2_FILES = ../deferred_processing_plus_transactions/images3.txt
```

Comment:
we only need 1 variable for all the deferred processing files

Reply:
They're used by 2 different repos...